RELEASE_PROCESS.md

```diff
@@ -0,0 +1,8 @@
+* Checkout master: `git checkout master`.
+* Get latest: `git pull --rebase origin master`
+* Bump version number in `lib/gnar-style/version.rb`.
+* Update the [CHANGELOG](CHANGELOG.md), moving the current release work to under the version number.
+* Create a commit, including the version number in the commit message: `git add
+  --all && git commit -m "Version x.y.z"`.
+* Push the release to RubyGems: `rake release`.
+* Check the RubyGems [twitter account](https://twitter.com/rubygems) to publicize the release.
```

Comment:
Are you suggesting that we publicize the release on Twitter or check the RubyGems account to verify it was released?

Reply:
That twitter account should automatically send out a tweet for new gem releases, but it looks like it hasn't been working since March 2. It makes for an easily-sharable source of information.

I'm not suggesting that part of the release process to be send out a tweet, or communicate with that account to ask them to send it out. It should be automated, but appears to be having an issue, or no longer be in use.

Reply:
If it's no longer in use or has an issue should we avoid confusion by removing this step?

Reply:
#36